target/bin/updatemailuser

```diff
@@ -10,25 +10,57 @@
 
 DATABASE=${DATABASE:-/tmp/docker-mailserver/postfix-accounts.cf}
 
-USER="${1}"
+CRYPTOKEY_UPDATE_PASSWORD=false
+CRYPTOKEY_GENERATE=false
+
+while getopts "cg" OPT
+do
+  case ${OPT} in
+    c ) CRYPTOKEY_UPDATE_PASSWORD="${3}" ;;
+    g ) CRYPTOKEY_GENERATE=true ;;
+    * )
+      echo "Invalid option: -${OPT}" >&2
+      ;;
+  esac
+done
+shift $(( OPTIND - 1 ))
```

Comment:
In fact his is a breaking change, isn't it? You can't use the script like you were used to to update a password

Reply:
How so? If no ops are passed in, the script runs just as it did before.

Reply:
Totally right, misread the `[OPTIONS]` and thought one need to specify `-c` or `-g` in any case now

Reply:
No worries! :)